tutorials/notebooks/pytorch/gptq/example_pytorch_mobilenetv2_cifar100_gptq.ipynb

```diff
@@ -0,0 +1,717 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "7cf96fb4",
+      "metadata": {
+        "id": "7cf96fb4"
+      },
+      "source": [
+        "# GPTQ - Pytorch MobileNetV2 on CIFAR100"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "59ed8f02",
+      "metadata": {
+        "id": "59ed8f02"
+      },
+      "source": [
+        "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/pytorch/gptq/example_pytorch_mobilenetv2_cifar100_gptq.ipynb)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "822944a1",
+      "metadata": {
+        "id": "822944a1"
+      },
+      "source": [
+        "## Overview"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "743dbc3d",
+      "metadata": {
+        "id": "743dbc3d"
+      },
+      "source": [
+        "This tutorial demonstrates the process of retraining and quantizing a MobileNetV2 on CIFAR100 dataset. It starts by fine-tuning a pretrained MobileNetV2 model on the CIFAR100 dataset. After retraining, the model is quantized using Gradient-Based PTQ (GPTQ) which uses unlabeled data for fine-tuning and improving the quantized model accuracy. This tutorial specifically uses mixed-precision quantization, which assigns different precision levels to different layers in the model based on their impact on the output. The quantized model is then evaluated and exported to an ONNX file."
+      ]
```

Comment:
"the model is quantized using Gradient-Based PTQ (GPTQ) which uses unlabeled data for fine-tuning" - I think it better to remove the "fine tuning" here since it can be confusing. (you mentioned fine tuning in the previous stage...). maybe: "weights adjustment "